### ISSUE TITLE
the comparison is not symmetric

### ISSUE THREAD

**Issue by paul2t** [davidreke/ChessRatingComparisonFrontEnd#50]
I got the following results:
A rating of 1900 in Chess.com: Rapid would equal 2110 in LiChess.org: Rapid, with an average variation of plus/minus of 110 points.
A rating of 2110 in LiChess.org: Rapid would equal 1778 in Chess.com: Rapid, with an average variation of plus/minus of 129 points.
Shouldn't it give me 1900 chess.com for 2110 lichess.org ?

**Comment by jxu** [comment]
Yes they should, although they are close enough to be within normal variation.
